test/validation/org/apache/cassandra/bridges/ArchiveClusterLogs.java

```diff
@@ -95,4 +105,62 @@ private static void compressFiles(ZipOutputStream zos,String folderName,String b
             throw new RuntimeException(e);
         }
     }
+
+    public static void savetempDirectoryPath(File CASSANDRA_DIR, File ccmDir)
+    {
+        File filePath = new File(CASSANDRA_DIR + "/build/test/logs/validation/tempDir.txt");
+
+        try
+        {
+            if(!filePath.exists())
+            {
+                filePath.getParentFile().mkdirs();
+            }
+            PrintWriter pw = new PrintWriter(new BufferedWriter(new FileWriter(filePath, true)));
+            pw.println(ccmDir);
+            pw.close();
+        }
+        catch (IOException e)
+        {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void removeOldCluster(File CASSANDRA_DIR)
```

Comment:
This method isn't re-usable, because it relies on calling CCMBridge specifically.